### DISCUSSION TITLE
i am facing an issue while i am generate a allure report via jenkins and jenkins running in ubuntu server and i using the chrome browser running on windows where i am accessing

### DISCUSSION THREAD

**Discussion by Dhianesh19** [allure-framework/allure2#2372] [Questions & Support]
![image (5)](https://github.com/allure-framework/allure2/assets/142984304/282f4d91-d448-4f56-844a-0d80f5800e90)
Now only our team start using the allure report for playwright javascript can you please help

**Comment by baev** [comment]
1. Please make sure you configured Allure Playwright integration [according to the docs](https://allurereport.org/docs/playwright/)
2. The `allure-results` folder should be generated after the tests run. The default location is project root, but you can change it by specifying `outputFolder` [reporter configuration option](https://allurereport.org/docs/playwright-configuration/#outputfolder)
3. Make sure the Jenkins job configuration has the correct path to `allure-results`.
